### ISSUE TITLE
Implement dynamics

### ISSUE THREAD

**Issue by ajyoon** [ajyoon/brown#6]
Dynamics should probably exist as a dedicated TextObject (or MusicGlyph?) class. There should be constructors for common accidentals, as well as a way to make custom dynamics with arbitrary text:

```python
Dynamic((Mm(30), staff.unit(7)), 'sfffz', staff)
Dynamic.p((Mm(30), staff.unit(7)), staff)
Dynamic.mf((Mm(30), staff.unit(7)), staff)
Dynamic.fff((Mm(30), staff.unit(7)), staff)
```

Dynamics should be drawn as multi-character text, combining available dynamics characters.
The `Dynamic` class should be able to map a-z characters to their appropriate SMuFL dynamic character canonical names. 

`Dynamic` should probably be implemented as a subclass of `MusicGlyph`. The fact that multiple characters will be used suggests that maybe the rigidity of the single-character `MusicGlyph` construction should be relaxed. Maybe the class should be renamed `MusicText`? 

See the [SMuFL spec page on dynamics](https://w3c.github.io/smufl/gitbook/tables/dynamics.html).

**Comment by ajyoon** [comment]
Multi-character `MusicText` seems like the way to go. See #1

**Comment by ajyoon** [comment]
Implemented in 0caa958ccd8cdb73021c86786e6b355552c70290
